### ISSUE TITLE
Train baseline + rotary embeddings

### ISSUE THREAD

**Issue by TevenLeScao** [bigscience-workshop/Megatron-DeepSpeed#42]
The simplest model, with rotary embeddings. Don't necessarily train to 300B tokens to compare.

**Comment by thomasw21** [comment]
Rotary embeddings is running similar to tr3:
 - scripts are here: https://github.com/bigscience-workshop/bigscience/pull/2
 - job_id: `708913`
 - user: `uty16tp`

**Comment by thomasw21** [comment]
Update: We've trying to fix some instabilities and re-run all experiments with:
 - batch size warmup.
 - increase learning rate.
 
 cc @TevenLeScao

**Comment by thomasw21** [comment]
Closing as we suspect that previous failure with rotary embeddings were linked to warmup
